tests/test_db_store.py

```diff
@@ -68,45 +68,45 @@ def test_insert_soil_moisture(self):
         """Should insert timestamp and moisture level into database."""
         record = db_store.SoilMoistureRecord(
             timestamp=datetime.datetime(
-                2016, 7, 23, 10, 51, 9, tzinfo=pytz.utc),
+                2016, 7, 23, 10, 51, tzinfo=pytz.utc),
```

Comment:
Oh I think the test insert timestamps should still go down to millisecond precision since that's how they get passed in from ```poller```.